### ISSUE TITLE
Multiple Vehicles

### ISSUE THREAD

**Issue by greggsymington** [dale3h/alexa-smartstart#2]
I have two vehicles in my SmartStart account. Is there a way to select which vehicle is works with? Any way to get it to work with both, one at a time?

**Comment by Planethill** [comment]
Yes!  This seems to be the only thing missing.  I also have two cars on one SmartStart account.  The only car I can control is first one that I installed a SmartStart device in.  Probably not a coincidence it's the default car.  I suppose I could create a second account for my wife's car and split the subscription, but having to do the the skills and everything twice seems excessive,   

This code is so close to being 100%,  I hope you find the time to work on it in 2019!
